.github/workflows/release.yml

```diff
@@ -28,9 +22,8 @@ jobs:
     outputs:
       version: ${{ inputs.version || github.ref_name }}
     steps:
-      - name: Ensure tag is from master
-        if: github.event_name == 'push' && github.event.base_ref != 'refs/heads/master'
```

Comment:
this was broken
`github.event.base_ref` was `null`.
I wanted this extra check, but for now I have no idea how to fix it and make it simple -- remove it for now